pallets/anmol-nft/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "anmol-nft"
```

Comment:
I thought this name was arbitrary enough to start with.

Reply:
I suppose one idea is to name it something more descriptive if we want to share the pallet further down the line. `fractionalizable-nft` or `extended-nft` or something.

Reply:
`anmol-nft` sounds good to me, @nblogist any thoughts?

Reply:
>The goal of this is to bring the NFT logic to a place where we can modify it and add logic

Given that this is supposed to be the general NFT pallet from where we extend other pallets `anmol-nft` sounds good but what would add clarity is  `base-nft`

But for fractionalized NFT `fractionalized-nft` would be good and convey what each pallet is meant to be, which can be extended from `base-nft` 

(base-nft suggested by Phaedra)

Reply:
Cool, I went and changed it to `base_nft` to better describe it.